src/plugins/messenger/MessengerPreview/components/MessengerListTemplate/components/MessengerListTemplateElement.tsx

```diff
@@ -44,43 +43,45 @@ export const getMessengerListTemplateElement = ({ React, styled }: MessagePlugin
 
 
     const MessengerListTemplateElement = ({ element, onAction, config }: IMessengerListTemplateElementProps) => {
-        const { title, subtitle, image_url, image_alt_text,  buttons, default_action } = element;
+        const { title, subtitle, image_url, image_alt_text, buttons, default_action } = element;
         // TODO default_action
 
         const button = buttons && buttons[0];
 
         const imgStyle: React.CSSProperties = {
-            backgroundImage: getBackgroundImage(image_url)
+            backgroundImage: getBackgroundImage(image_url),
+            width: '25%'
         }
         const messengerSubtitleId = useRandomId("webchatListTemplateSubtitle");
         const messengerTitle = title ? title + ". " : "";
-        const messengerAriaLabel  = default_action?.url ? messengerTitle + "Opens in new tab" : title;
+        const messengerAriaLabel = default_action?.url ? messengerTitle + "Opens in new tab" : title;
 
         const handleKeyDown = (event, default_action) => {
-            if(event.key === "Enter" && default_action) {
+            if (event.key === "Enter" && default_action) {
                 onAction(event, default_action);
             }
         }
 
         return (
             <div role="listitem">
                 <Root
-                    role={default_action?.url ? "link" : undefined} 
+                    role={default_action?.url ? "link" : undefined}
                     onClick={default_action && (e => onAction(e, default_action))}
+                    style={default_action?.url ? { cursor: "pointer" } : {}}
```

Comment:
Merge regression: This inline style can be removed. The same style is applied using class 'link'.

Reply:
fixed